content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Shared control plane (single and multiple networks)
-description: Install an Istio mesh across multiple Kubernetes clusters with a shared control plane.
+title: Deploy Remote Clusters on Single and Multiple networks
```

Comment:
This page is not solely about setting up remote clusters. It's also describes how to setup the primary cluster. It's divided into two parts:

1. Add primary cluster to a mesh (cluster with control plane)
2. Add remote cluster to a mesh (cluster using an external control plane)

... where (1) and (2) can be on the same/different network. The user could repeat (1) to create a mesh with replicated control planes, though we don't document that usage here. 

In the future (not this PR), does it make sense to have a separate page just for adding a remote cluster? Then we'd have something like the following:

1. How to install Istio and create a mesh 
2. How to add a second cluster to a mesh.
2a. Using control plane from another cluster (i.e. remote cluster)
2b. Using control plane running in the second cluster (i.e. primary cluster)
3. Something about multi-mesh (for example, how to federate meshes)

Single/Multi-network would be options with 2a and 2b. 

We'd also need to consider how central istiod instructions fit in. For example, does it get its own page, or does it get included in (1)? cc @nmittler @adammil2000 @linsun

Reply:
+1, that's exactly how I would like to see it. I think it's critical that the setup for multicluster (step 2) should be separate instructions/docs that start from any existing Istio installation. There should not be a requirement to plan for multicluster on day 1, but last time we had this discussion, there were still some implementation details that needed to be different when the first Istio cluster is set up depending on whether or not it was going to be part of a multicluster config or not, and IIRC it couldn't be switched later. Maybe that has since been fixed, which would definitely encourage us to move to a doc structure like above, asap.

Reply:
@ayj This PR is not about any of that. This PR is just implementing the new terminology on existing content.
To review the content tackling those tasks, please see: https://docs.google.com/document/d/1wzoFqLHw-dEoKPVi1w5tlckWTn78qpYNbC5fEwp-Mw4/edit
I'm working on the PR for that content now and you will have the ability to review it as soon as can get it submitted.
@ayj I would also appreciate if you could keep the scope of your review to the content changes in the PR to ensure we can make progress iteratively.

Reply:
@ayj Your general point is fine and we can discuss that in a future PR when we add more of the "lego" content for multi-network, etc. For now, I think the only action here is to change the title to something like "Configure Primary/Remote clusters on Single and Multiple Networks".

Once we have all the content moved over to lego form, this doc will simply go away, since it will be redundant.

Reply:
@nmittler's suggestion SGTM. 

@rcaballeromx - sorry for the derail and thanks for the link to the tracking :) I'll add any content related comments there instead.